functions/zoom-webhook.py

```diff
@@ -130,6 +130,9 @@ def validate_payload(payload):
                     .format(field, obj.keys()))
 
         files = obj["recording_files"]
+        if not len(files):
+            raise BadWebhookData("No recording files in request payload")
+
```

Comment:
Move this check to after the files are filtered by file type.

Reply:
Now it's only confirming that any files are mp4